### DISCUSSION TITLE
fast probing with bltouch

### DISCUSSION THREAD

**Discussion by roliam56** [Jyers/Marlin#803] [General Discussion]
Hello

in firmware  tititopher68 , In which configuration file or setting is the rapid bed setting located,  without raising the probe between several points?

 Thanks, and great job, this is the best firmware I have found, great !!

**Comment by Makney** [comment]
I think you mean the HS Mode which is ` #define BLTOUCH_HS_MODE` in Configuration_adv.h
or do you mean something else?

**Reply by roliam56** [reply]
that's exactly what I'm looking for, quick answer !!  thank you
